easybuild/easyconfigs/c/CMake/CMake-3.11.4-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.11.4'
+
+homepage = 'http://www.cmake.org'
+
+description = """
+ CMake, the cross-platform, open-source build system.  CMake is a family of
+ tools designed to build, test and package software.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['8f864e9f78917de3e1483e256270daabc4a321741592c5b36af028e72bff87f5']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('ncurses', '6.1'),
```

Comment:
@Falkor I only noticed this after merging, but using `ncurses` 6.1 here severely limits the usefulness of these PRs, since in both the `*/2017b` and `*/2018a` toolchain generation easyconfigs still use `ncurses` 6.0

So, as soon as any dependency that loads `ncurses` 6.0 is used, these `CMake` easyconfigs can't be used as a dependency.

Any objections against downgrading these `ncurses` versions to `6.0`?

For a CMake built with `GCCcore/7.3.0` (i.e. the base of the `*/2018b` toolchains), going with `ncurses` 6.1 makes perfect sense though...

Reply:
@Falkor Change made in #6765, please give feedback there

Reply:
Sure no problem. I'm checking the successful builds now on our cluster, if that's fine I'll commit the changed versions to that branch.

Reply:
Indeed, everything went fine so I pushed it

Reply:
Weird, I cannot see the push in `feature/cmake-3.11.4_3.12.1` anymore... Should I reopen a pull request ?

Reply:
got it, sorry for the mess here then.